async-request-reply/deploy.bicep

```diff
@@ -42,75 +117,164 @@ resource dataStorageAccount 'Microsoft.Storage/storageAccounts@2019-04-01' = {
+    //virtualNetworkSubnetId: functiondsbnt.id // Adding VNet Integration
     siteConfig: {
+      netFrameworkVersion: 'v8.0'
+      numberOfWorkers: 1
+      alwaysOn: false
+      http20Enabled: false
+      minimumElasticInstanceCount: 0
+      use32BitWorkerProcess: false
       appSettings: [
         {
-          name: 'AzureWebJobsDashboard'
-          value: 'DefaultEndpointsProtocol=https;AccountName=${appStorageAccount.name};AccountKey=${listKeys(appStorageAccount.id, '2015-05-01-preview').key1}'
-        }
-        {
-          name: 'AzureWebJobsStorage'
-          value: 'DefaultEndpointsProtocol=https;AccountName=${appStorageAccount.name};AccountKey=${listKeys(appStorageAccount.id, '2015-05-01-preview').key1}'
-        }
-        {
-          name: 'WEBSITE_CONTENTAZUREFILECONNECTIONSTRING'
-          value: 'DefaultEndpointsProtocol=https;AccountName=${appStorageAccount.name};AccountKey=${listKeys(appStorageAccount.id, '2015-05-01-preview').key1}'
-        }
-        {
-          name: 'WEBSITE_CONTENTSHARE'
-          value: toLower(functionAppName)
+          name: 'APPLICATIONINSIGHTS_CONNECTION_STRING'
+          value: applicationInsights.properties.ConnectionString
         }
         {
-          name: 'FUNCTIONS_EXTENSION_VERSION'
-          value: '~4'
+          name: 'AzureWebJobsStorage__accountName'
+          value: appStorageAccount.name
         }
         {
-          name: 'WEBSITE_NODE_DEFAULT_VERSION'
-          value: '8.11.1'
+          name: 'ServiceBusConnection__fullyQualifiedNamespace'
+          value: '${serviceBusNamespace.name}.servicebus.windows.net'
         }
         {
-          name: 'FUNCTIONS_WORKER_RUNTIME'
-          value: 'dotnet'
-        }
-        {
-          name: 'ServiceBusConnectionAppSetting'
-          value: listKeys(resourceId('Microsoft.ServiceBus/namespaces/AuthorizationRules', serviceBusNamespace.name, 'RootManageSharedAccessKey'), '2015-08-01').primaryConnectionString
-        }
-        {
-          name: 'StorageConnectionAppSetting'
-          value: 'DefaultEndpointsProtocol=https;AccountName=${dataStorageAccount.name};AccountKey=${listKeys(dataStorageAccount.id, '2015-05-01-preview').key1}'
+          name: 'DataStorage__blobServiceUri'
+          value: 'https://${dataStorageAccount.name}.blob.${environment().suffixes.storage}'
         }
       ]
     }
+    clientCertEnabled: true
+    clientCertMode: 'OptionalInteractiveUser'
+    clientCertExclusionPaths: '/public'
+    functionAppConfig: {
+      deployment: {
+        storage: {
+          type: 'blobContainer'
+          value: '${appStorageAccount.properties.primaryEndpoints.blob}${deploymentStorageContainerName}'
+          authentication: {
+            type: 'SystemAssignedIdentity'
+          }
+        }
+      }
+      scaleAndConcurrency: {
+        maximumInstanceCount: 100
+        instanceMemoryMB: 2048
+      }
+      runtime: {
+        name: 'dotnet-isolated'
+        version: '8.0'
```

Comment:
<3

Reply:
Error deploying.
"The specified value of runtime version '<3' for runtime name 'dotnet-isolated' is not supported. Please set it to one of the allowed values: 8.0"
I will keep 8.0

Reply:
LOL!!!! 

"<3" is a heart.  I'm indicating that I'm glad you made this change.

Reply:
<3 = ❤️